src/libcore/slice.rs

```diff
@@ -915,6 +917,9 @@ macro_rules! make_mut_slice {
 ///     println!("{}", element);
 /// }
 /// ```
+///
+/// [`iter`]: ../primitive.slice.html#method.iter
```

Comment:
These need to be like `../../std/primitive.slice.html#method.iter` so the links work in the `core` docs as well.